src/bin/client-demo.rs

```diff
@@ -13,22 +13,25 @@ use std::thread::sleep;
 use std::time::{Duration, Instant};
 
 fn main() {
-    let addr = "127.0.0.1:8000";
-    let send_addr = "127.0.0.1:8001";
+    let addr = "127.0.0.1:9000";
```

Comment:
Can you find another port? This is the same port the skel test uses. Using it here means you can't leave `cargo watch` running while playing with the demo.